### PR TITLE
add `last_updated_time_in_index` to `retrieve_dataframe`

### PR DESCRIPTION
**Pull request by KristianVangsnes**
## Description
Adding last_updated_time_in_index to retrieve_dataframe function, which makes index into multi-index. Also updated docstring.

## Checklist:
- [x] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Review by haakonvt — COMMENTED** [review]
LGTM! Let's add a test making sure it works when `len(rows)` is 0, 1, and many 👌

**Comment by KristianVangsnes** [comment]
Where would be best to add the test? tests_unit/test_api/test_raw.py?

**Comment by haakonvt** [comment]
> Where would be best to add the test? tests_unit/test_api/test_raw.py?

Yup!

File: cognite/client/_api/raw.py [hunk 1]
```diff
@@ -756,7 +756,9 @@ def retrieve_dataframe(
         pd = local_import("pandas")
         rows = self.list(db_name, table_name, min_last_updated_time, max_last_updated_time, columns, limit, partitions)
         if last_updated_time_in_index:
-            idx = pd.MultiIndex.from_tuples([(r.key, r.last_updated_time) for r in rows], names=["key", "last_updated_time"])
+            idx = pd.MultiIndex.from_tuples(
+                [(r.key, r.last_updated_time) for r in rows], names=["key", "last_updated_time"]
+            )
```
Comment (haakonvt) [inline cognite/client/_api/raw.py:761]: Should we convert epoch-ms to datetime?

File: tests/tests_unit/test_api/test_raw.py [hunk 2]
```diff
@@ -64,6 +64,47 @@ def mock_retrieve_raw_row_response(rsps, cognite_client):
...
+        rsps.GET,
+        cognite_client.raw._get_base_url_with_base_path() + "/raw/dbs/db1/tables/table1/rows",
+        status=200,
+        json=response_body,
+    )
+    yield rsps
+
+
+@pytest.fixture
+def mock_retrieve_raw_rows_response_one_rows(rsps, cognite_client):
```
Comment (haakonvt) [inline tests/tests_unit/test_api/test_raw.py:85]: 😅 
```suggestion
def mock_retrieve_raw_rows_response_one_row(rsps, cognite_client):
```

File: cognite/client/_api/raw.py (hunk 1 above)
Comment (KristianVangsnes) [reply cognite/client/_api/raw.py:761]: Good questions, I don't see why not.

**Review by haakonvt — APPROVED** [review]
LGTM! Might need to update one or more tests after making it a timestamp? Also, please try to revert the unrelated docstring changes (unless you actually fixed something in them 👌 )

**Pull request merged**